### PR TITLE
chore: [#186659137] remove style prop from unstyled button

### PR DESCRIPTION
**Pull request by FarazA22**
### Ticket

[Replace with ticket_id](https://www.pivotaltracker.com/story/show/)

## Code author checklist

- [ ] I have performed a self-review of my code
- [ ] I have created and/or updated relevant documentation (EX: [Engineering Reference/FAQ](https://docs.google.com/document/d/1X4iWSGmBZdHYZ0jGqkwTR3_yyyqI_TiiJptfc8pi9Po)), if necessary
- [ ] I have not used any relative imports
- [ ] I have checked for and removed instances of unused config from CMS
- [ ] I have pruned any instances of unused code
- [ ] I have not added any markdown to labels, titles and button text in config
- [ ] If I added/updated any values in `userData` (including `profileData`, `formationData` etc), then I added a new migration file
- [ ] If I added any new collections to the CMS config, then I updated the search tool and `cmsCollections.ts`

### CODE REVIEW

File: web/src/components/PageFooter.tsx [hunk 1]
```diff
@@ -48,17 +46,16 @@ export const PageFooter = (): ReactElement => {
           <hr className="margin-y-2" />
           <div className={`${isLargeScreen ? "" : "flex flex-column flex-align-center"}`}>
             <span className="padding-right-1 text-center">
-              <UnStyledButton
-                onClick={(): void => openInNewTab(Config.footer.officeLink)}
-                className="text-white text-center"
-                ariaLabel="Made with love by the Office of Innovation"
-                style="footerLink"
-                isUnderline={true}
+              <a
```
Comment (AnJuHyppolite) [inline web/src/components/PageFooter.tsx:49]: [boulder] Can you provide context as to why we're removing this? This recent update was done to make "Made with ❤️..." accessible.

File: web/src/components/njwds-extended/UnStyledButton.tsx [hunk 2]
```diff
@@ -1,34 +1,24 @@
 import React, { forwardRef, ReactElement, useEffect, useRef, useState } from "react";
 
 interface Props {
-  style: "default" | "transparentBgColor" | "footerLink";
   className?: string;
```
Comment (AnJuHyppolite) [inline web/src/components/njwds-extended/UnStyledButton.tsx:4]: [boulder] How are we accounting for the footer?

File: web/src/components/njwds-extended/UnStyledButton.tsx [hunk 3]
```diff
@@ -42,14 +32,12 @@ export const UnStyledButton = forwardRef(
       }
     }, [height, width, disabledClass]);
 
-    const noRightMargin =
```
Comment (AnJuHyppolite) [inline web/src/components/njwds-extended/UnStyledButton.tsx:45]: [boulder] This change affects the the Primary and Secondary buttons. Are the style changes affecting these other two components in a way that we want them displayed?

**Review by AnJuHyppolite — COMMENTED** [review]
@FarazA22 Can you update the Tracker link so it takes us to the story? Also, would love some context around why we're removing the style prop in the Tracker ticket itself and here in the description.

File: web/src/components/njwds-extended/UnStyledButton.tsx (hunk 3 above)
Comment (FarazA22) [reply web/src/components/njwds-extended/UnStyledButton.tsx:45]: Based on how the logic was written, the no right margin was applied to every button except the "help" formation button. I've corrected that padding.

File: web/src/components/njwds-extended/UnStyledButton.tsx (hunk 2 above)
Comment (FarazA22) [reply web/src/components/njwds-extended/UnStyledButton.tsx:4]: it's converted to an anchor link. Defining styles by location of the app was a motivating factor behind the refactor; I don't believe footerLink accurately describes what the button is doing if selected.

File: web/src/components/PageFooter.tsx (hunk 1 above)
Comment (FarazA22) [reply web/src/components/PageFooter.tsx:49]: an anchor link should be used since it's an external link

**Pull request merged**